indexFinal.html

```diff
@@ -5,53 +5,49 @@
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <title>Document</title>
-        <link rel="stylesheet" href="stylesheetFinal.css"/>
+        <link rel="stylesheet" href="stylesheetFinal1.css"/>
```

Comment:
Whay did you rename the stylesheet. There is no `stylesheetFinal1` in your project.

Now it looks like this
<img width="1001" alt="image" src="https://user-images.githubusercontent.com/36345162/208043812-f6b319ca-ea5b-4df3-88e5-ed9cd9f6e25a.png">